docling-serve/docling-serve-api/src/main/java/ai/docling/serve/api/DoclingServeConvertApi.java

```diff
@@ -9,11 +11,35 @@
  * a structured or processed document format based on the specified conversion options.
  */
 public interface DoclingServeConvertApi {
-    /**
+  /**
    * Converts the provided document source(s) into a processed document based on the specified options.
    *
    * @param request the {@link ConvertDocumentRequest} containing the source(s), conversion options, and optional target.
    * @return a {@link ConvertDocumentResponse} containing the processed document data, processing details, and any errors.
    */
   ConvertDocumentResponse convertSource(ConvertDocumentRequest request);
+
+  /**
+   * Initiates an asynchronous conversion of the provided document source(s) and returns a
+   * {@link CompletableFuture} that completes when the conversion is done.
+   *
+   * <p>This method starts the conversion, polls the status in the background, and completes
+   * the future with the result when the conversion finishes. The polling happens on a separate
```

Comment:
Perhaps we should remove this final sentence as it's up to the framework/client to configure how the polling happens (platform thread, virtual thread, some other mechanism provided by Mutiny/Reactor). Thoughts?